zerver/tests/test_tex.py

```diff
@@ -0,0 +1,12 @@
+import mock
+from django.conf import settings
+
+from zerver.lib.tex import render_tex
+from zerver.lib.test_classes import ZulipTestCase
+
+class RenderTexTest(ZulipTestCase):
+    def test_invalid_path(self) -> None:
+        with self.settings(STATIC_ROOT="/invalid/path"):
+            with mock.patch('logging.error') as mock_logger:
+                render_tex("random text")
+                mock_logger.assert_called_with("Cannot find KaTeX for latex rendering!")
```

Comment:
I'd put this test function inside `test_bugdown.py`, since it's part of the markdown processor, and we really don't need a new file for a single test.